topics/metagenomics/tutorials/beer-data-analysis/tutorial.md

```diff
@@ -0,0 +1,558 @@
+---
+layout: tutorial_hands_on
+
+title: Identification of yeasts in a beer using Nanopore sequencing
+zenodo_link: https://doi.org/10.5281/zenodo.6620778
```

Comment:
Reupload data to Zenodo

Reply:
The new Zenodo version: [10.5281/zenodo.7093173](https://zenodo.org/badge/DOI/10.5281/zenodo.7093173.svg)